src/Evolution/DiscontinuousGalerkin/Actions/ComputeTimeDerivative.hpp

```diff
@@ -1026,58 +1029,111 @@ template <typename ParallelComponent, typename DbTagsList>
 void ComputeTimeDerivative<Metavariables>::send_data_for_fluxes(
     const gsl::not_null<Parallel::GlobalCache<Metavariables>*> cache,
     const db::DataBox<DbTagsList>& box) noexcept {
-  using BoundaryScheme = typename Metavariables::boundary_scheme;
-  static constexpr size_t volume_dim = BoundaryScheme::volume_dim;
-  using temporal_id_tag = typename BoundaryScheme::temporal_id_tag;
-  using receive_temporal_id_tag =
-      typename BoundaryScheme::receive_temporal_id_tag;
-  using fluxes_inbox_tag = ::dg::FluxesInboxTag<BoundaryScheme>;
-  using all_mortar_data_tag =
-      ::Tags::Mortars<typename BoundaryScheme::mortar_data_tag, volume_dim>;
-
+  using system = typename Metavariables::system;
+  constexpr size_t volume_dim = Metavariables::volume_dim;
   auto& receiver_proxy =
       Parallel::get_parallel_component<ParallelComponent>(*cache);
-
-  const auto& all_mortar_data = get<all_mortar_data_tag>(box);
   const auto& element = db::get<domain::Tags::Element<volume_dim>>(box);
-  const auto& temporal_id = db::get<temporal_id_tag>(box);
-  const auto& receive_temporal_id = db::get<receive_temporal_id_tag>(box);
-  const auto& mortar_meshes =
-      db::get<::Tags::Mortars<domain::Tags::Mesh<volume_dim - 1>, volume_dim>>(
-          box);
 
-  // Iterate over neighbors
-  for (const auto& [direction, neighbors] : element.neighbors()) {
-    const size_t dimension = direction.dimension();
-    const auto& orientation = neighbors.orientation();
-    const auto direction_from_neighbor = orientation(direction.opposite());
+  if constexpr (detail::has_boundary_correction_v<system>) {
+    const auto& time_step_id = db::get<::Tags::TimeStepId>(box);
+    const auto& all_mortar_data =
+        db::get<evolution::dg::Tags::MortarData<volume_dim>>(box);
+
+    for (const auto& [direction, neighbors] : element.neighbors()) {
+      const auto& orientation = neighbors.orientation();
+      const auto direction_from_neighbor = orientation(direction.opposite());
+
+      for (const auto& neighbor : neighbors) {
+        const std::pair mortar_id{direction, neighbor};
+
+        std::pair<Mesh<volume_dim - 1>, std::vector<double>>
+            remote_boundary_data_on_mortar =
```

Comment:
For consistency with an earlier PR in this series, should "remote" -> "neighbor" ?